frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -195,186 +322,87 @@ export function Experiment(): JSX.Element {
-                                        <ul>
-                                            {newExperimentData?.filters?.properties?.length ? (
-                                                newExperimentData.filters.properties.map(
-                                                    (property: PropertyFilter, idx: number) => (
-                                                        <li key={idx}>
-                                                            Users with {property.key} {property.operator}{' '}
-                                                            {Array.isArray(property.value)
-                                                                ? property.value.map((val) => `${val}, `)
-                                                                : property.value}
-                                                        </li>
-                                                    )
-                                                )
-                                            ) : (
-                                                <li key={'all users'}>All users</li>
-                                            )}
-                                        </ul>
-                                    </Col>
-                                </Row>
-                                <Row>
-                                    <Col>
-                                        <Row>Experiment parameters:</Row>
-                                        <Row>
-                                            <ul>
-                                                <li>Target confidence level: </li>
-                                                <li>Approx. run time: </li>
-                                                <li>Approx. sample size: </li>
-                                            </ul>
-                                        </Row>
-                                    </Col>
-                                </Row>
-                                <Row justify="space-between">
-                                    <Button onClick={prevPage}>Go back</Button>
-                                    <Button type="primary" onClick={() => createExperiment()}>
-                                        Save and launch
-                                    </Button>
-                                </Row>
+                                <Button
+                                    icon={<SaveOutlined />}
+                                    className="float-right"
+                                    type="primary"
+                                    htmlType="submit"
+                                >
+                                    Save
+                                </Button>
                             </div>
-                        )}
+                        </div>
                     </Form>
                 </>
+            ) : !experimentData?.start_date ? (
+                <div className="confirmation">
```

Comment:
Why do we need the confirmation page to check a duplicate feature flag key? We can do that in the form on save 

https://user-images.githubusercontent.com/25164963/146389991-dd92bbb3-ec4d-4df2-92a5-7adee318002b.mov

Reply:
The flow is like this:

1. Save Experiment, which creates a FF.
2. Now, go to your codebase, use this FF, test that your code works with the variants on this FF
3. Now, launch experiment

We need the "Experiment summary screen" (a.k.a. confirmation screen) to allow step 2 to happen. We can't directly launch the experiment on save, because that's too early: the FF has just been created, there's no code / nothing that works with the control/test variants. (or atleast not tested)

Reply:
OK so basically what we want from here is to set the user up for success to avoid empty results?

Reply:
Set them up for success, yes!

re: avoiding empty results - not quite, since the experiment start_date is when launch is pressed, initially data is always empty.

Good point there actually:

@clarkus : When the experiment starts, there will be no data. It takes a while to gather data. We want to keep this in mind for the results screen! (and have good initial prompts which explain there's nothing wrong, just come back later)